web/src/pages/BookWizard/OverviewPage.module.css

```diff
@@ -127,5 +127,134 @@
 .buttonContainer {
   text-align: center;
 }
-  
-  
\ No newline at end of file
+
+.pageContainer {
+  display: flex;
+  justify-content: space-evenly;
+  margin-top: 50px;
+  margin-bottom: 50px;
+}
+
+.tabContainer {
+  width: 18em;
+  height: 30em;
+  overflow-y: scroll;
+  border: 1px solid var(--dark-grey);
+  margin-left: 20px;
+  margin-right: 20px;
+}
+
+.imageContainer {
+  display: block;
+  width: 12em;
+  margin: 30px auto;
+  box-shadow: -1px -1px 16px 4px var(--light-grey);
+}
+
+.coverImage {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
+
+.bookTitle {
+  text-align: center;
+  font-family: 'GothamRounded - Bold';
+  font-size: 1.5em;
+  margin-bottom: 6px;
+}
+
+.bookAuthor {
+  text-align: center;
+  font-family: 'GothamRounded - Light';
+  font-size: 1em;
+  margin-bottom: 25px;
+}
+
+.buttonGroup {
+  display: flex;
+  justify-content: space-evenly;
+  margin-left: 20px;
+  margin-right: 20px;
+  margin-bottom: 20px;
+}
+
+.tabButtonActive {
+  width: 3em;
+  height: 1em;
+  background-color: var(--orange);
+  border-radius: 5px;
+  border-style: solid;
+  border-color: var(--orange);
+  border: 1px solid var(--orange);
+  padding-top: 3px;
+  box-shadow: 2px 2px 8px var(--light-grey);
+}
+
+.tabButtonInactive {
+  width: 3em;
+  height: 1em;
+  border-radius: 5px;
+  border-style: solid;
+  border-color: var(--orange);
+  border: 1px solid var(--orange);
+  padding-top: 3px;
+  box-shadow: 2px 2px 8px var(--light-grey);
+}
+
+.buttonActive {
+  text-align: center;
+  font-size: 0.7em;
+  font-family: 'GothamRounded - Bold';
+  color: var(--white);
+}
+
+.buttonInactive {
+  text-align: center;
+  font-size: 0.7em;
+  font-family: 'GothamRounded - Bold';
+  color: var(--orange);
+}
+
+.videoContainer {
+  margin-left: auto;
+  margin-right: auto;
+  width: 90%;
+}
+
+.video {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+  margin-bottom: 10px;
+}
+
+.markdownText {
+  font-size: 0.6em;
```

Comment:
Yeah that did work, but IMO 0.6 is still a bit too small? 0.8 looked good on my screen and in 1080p, thoughts?

Reply:
![Screen Shot 2021-12-01 at 10 04 59 PM](https://user-images.githubusercontent.com/47440327/144366968-b5a48be4-dcb2-4ea3-bcd8-6efaab94e254.png)

0.8 looks very large on my screen. Should we compromise and go with 0.7?